isis/tests/FunctionalTestsJigsaw.cpp

```diff
@@ -526,98 +526,107 @@ TEST_F(ObservationPair, FunctionalTestJigsawCamSolveAll) {
-  EXPECT_NEAR(zstats.Maximum(),           793.9672179283682,  0.00001); 
+  EXPECT_NEAR(zstats.Maximum(),           793.9672179283682,  0.00001);
 
-  Camera *cam = cubeL->camera(); 
+  Camera *cam = cubeL->camera();
   SpiceRotation *rot = cam->instrumentRotation();
-  std::vector<double> a1; 
-  std::vector<double> a2; 
+  std::vector<double> a1;
+  std::vector<double> a2;
   std::vector<double> a3;
 
-  rot->GetPolynomial(a1, a2, a3); 
-  
+  rot->GetPolynomial(a1, a2, a3);
+
   EXPECT_NEAR(a1.at(0), 2.16338,    0.0001);
   EXPECT_NEAR(a1.at(1), -0.0264475, 0.0001);
   EXPECT_NEAR(a1.at(2), 0.00469675, 0.0001);
   EXPECT_NEAR(a1.at(3), 0.0210955,  0.0001);
-  
+
   EXPECT_NEAR(a2.at(0), 1.83011,     0.0001);
   EXPECT_NEAR(a2.at(1), -0.0244244,  0.0001);
   EXPECT_NEAR(a2.at(2), -0.00456569, 0.0001);
   EXPECT_NEAR(a2.at(3), 0.00637157,  0.0001);
 
-  QFile file(prefix.path() + "/bundleout_images.csv");
-  if (!file.open(QIODevice::ReadOnly)) {
-    FAIL() << file.errorString().toStdString();
+  QFile leftFile(prefix.path() + "/bundleout_images_LUNARRECONNAISSANCEORBITER_NACL.csv");
+  if (!leftFile.open(QIODevice::ReadOnly)) {
+    FAIL() << leftFile.errorString().toStdString();
   }
-  
-  // skip the first two lines, we don't want to compare the header. 
-  file.readLine();
-  file.readLine(); 
-
-  QString line = file.readLine();
-  QStringList elems = line.split(","); 
-  
+
+  // skip the first two lines, we don't want to compare the header.
+  leftFile.readLine();
+  leftFile.readLine();
+
+  QString line = leftFile.readLine();
+  QStringList elems = line.split(",");
+
   // RA(t0) final
-  EXPECT_NEAR(elems.at(21).toDouble(), 123.9524918, 0.00001); 
+  EXPECT_NEAR(elems.at(21).toDouble(), 123.9524918, 0.00001);
   // RA(t1) final
-  EXPECT_NEAR(elems.at(26).toDouble(), -1.51532975, 0.00001); 
+  EXPECT_NEAR(elems.at(26).toDouble(), -1.51532975, 0.00001);
   // RA(t2) final
-  EXPECT_NEAR(elems.at(31).toDouble(), 0.2691039,   0.00001); 
+  EXPECT_NEAR(elems.at(31).toDouble(), 0.2691039,   0.00001);
   // RA(t3) final
-  EXPECT_NEAR(elems.at(36).toDouble(), 1.208684781, 0.00001); 
+  EXPECT_NEAR(elems.at(36).toDouble(), 1.208684781, 0.00001);
 
   // DEC(t0) final
-  EXPECT_NEAR(elems.at(41).toDouble(), 104.8575294,       0.00001); 
+  EXPECT_NEAR(elems.at(41).toDouble(), 104.8575294,       0.00001);
   // DEC(t1) final
-  EXPECT_NEAR(elems.at(46).toDouble(), -1.399416621,      0.00001); 
+  EXPECT_NEAR(elems.at(46).toDouble(), -1.399416621,      0.00001);
   // DEC(t2) final
-  EXPECT_NEAR(elems.at(51).toDouble(), -0.26159502200533, 0.00001); 
+  EXPECT_NEAR(elems.at(51).toDouble(), -0.26159502200533, 0.00001);
   // DEC(t3) final
-  EXPECT_NEAR(elems.at(56).toDouble(), 0.365064224,       0.00001); 
-  
-  
-  line = file.readLine();
-  elems = line.split(","); 
-  
+  EXPECT_NEAR(elems.at(56).toDouble(), 0.365064224,       0.00001);
+
+
+  QFile rightFile(prefix.path() + "/bundleout_images_LUNARRECONNAISSANCEORBITER_NACR.csv");
```

Comment:
This test is an example of how the new images.csv output could be annoying. The NACL and NACR are two copies of the same system and we would solve for their images exactly the same way, but because they are actually different instrument IDs, we will output two images.csv files when we don't need to.

I want to talk to some users to see how they feel about this change.